Domain_classes/Account.py

```diff
@@ -1,10 +1,14 @@
+
 class Account():
-    def __init__(self, account_id,name,open_transaction,balance):
+    def __init__(self, account_id,name,balance:int, open_transaction=None):
```

Comment:
Czy python mamy typ double aktualnie balance to int czyli liczba całkowita a przecież stan konta jest zmienno przecionkowy

Reply:
racja, poprawione